coming-soon/Pages/_ViewStart.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿@{
```

Comment:
Is this file used anywhere?

Reply:
i think it is, by convention, everywhere - though obviously the layout choice can be overridden:
https://www.c-sharpcorner.com/blogs/layouts-and-viewstartcshtml-in-mvc#:~:text=_Viewstart.,be%20rendered%20along%20with%20it.